jsk_apc2016_common/node_scripts/visualize_pick_json.py

```diff
@@ -11,6 +11,9 @@
 
 
 def publish_cb(event):
+    json = rospy.get_param('~json', None)
+    img = jsk_apc2016_common.visualize_pick_json(json)
+    imgmsg = br.cv2_to_imgmsg(img, encoding='bgr8')
```

Comment:
They are necessary.

rosparam `json` gets dynamically changed. Output of this node has to reflect the change.